examples/child_window.rs

```diff
@@ -13,52 +6,53 @@ mod fill;
 fn main() -> Result<(), impl std::error::Error> {
     use std::collections::HashMap;
 
+    #[path = "util/fill.rs"]
+    mod fill;
+
     use winit::{
         dpi::{LogicalPosition, LogicalSize, Position},
         event::{ElementState, Event, KeyEvent, WindowEvent},
-        event_loop::{EventLoop, EventLoopWindowTarget},
+        event_loop::{ActiveEventLoop, EventLoop},
         raw_window_handle::HasRawWindowHandle,
         window::{Window, WindowId},
     };
 
-    fn spawn_child_window(
-        parent: &Window,
-        event_loop: &EventLoopWindowTarget,
-        windows: &mut HashMap<WindowId, Window>,
-    ) {
+    fn spawn_child_window(parent: &Window, event_loop: &ActiveEventLoop) -> Window {
         let parent = parent.raw_window_handle().unwrap();
-        let mut builder = Window::builder()
+        let mut window_attributes = Window::attributes()
             .with_title("child window")
             .with_inner_size(LogicalSize::new(200.0f32, 200.0f32))
             .with_position(Position::Logical(LogicalPosition::new(0.0, 0.0)))
             .with_visible(true);
         // `with_parent_window` is unsafe. Parent window must be a valid window.
-        builder = unsafe { builder.with_parent_window(Some(parent)) };
-        let child_window = builder.build(event_loop).unwrap();
+        window_attributes = unsafe { window_attributes.with_parent_window(Some(parent)) };
 
-        let id = child_window.id();
-        windows.insert(id, child_window);
-        println!("child window created with id: {id:?}");
+        event_loop.create_window(window_attributes).unwrap()
     }
 
     let mut windows = HashMap::new();
 
     let event_loop: EventLoop<()> = EventLoop::new().unwrap();
-    let parent_window = Window::builder()
-        .with_title("parent window")
-        .with_position(Position::Logical(LogicalPosition::new(0.0, 0.0)))
-        .with_inner_size(LogicalSize::new(640.0f32, 480.0f32))
-        .build(&event_loop)
-        .unwrap();
+    let mut parent_window_id = unsafe { WindowId::dummy() };
```

Comment:
Nit: Let's try to avoid `unsafe` in examples when we can.

```suggestion
    let mut parent_window_id = None;
```